File: clear-all/clear-all/src/main/java/com/example/clear_all/dto/response/LeaguesQuery.java
package com.example.clear_all.dto.response;


import lombok.Data;

import java.math.BigDecimal;
import java.math.BigInteger;
import java.util.Date;

@Data
public class LeaguesQuery {
    private BigDecimal id;
    private String leagueName;
    private String regionCountry;
    private Short division;
    private Date seasonStart;
    private Date seasonEnd;
    private String logoUrl;
    private BigInteger createBy;
    private Date createDate;
    private BigInteger lastUpdateBy;
    private Date lastUpdateDate;
}

File: WebSport24hNews/Application/Query/Model/24hLeagues/LeaguesQuery.cs
using Microsoft.EntityFrameworkCore;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace WebSport24hNews.Application.Query.Model._24hLeagues
{
    public class LeaguesQuery
    {
        public decimal Id { get; set; }
        public string? LeagueName { get; set; }

        public string? RegionCountry { get; set; }

        public bool? Division { get; set; }
        public DateTime? SeasonStart { get; set; }

        public DateTime? SeasonEnd { get; set; }
        public string? LogoUrl { get; set; }

        public decimal CreateBy { get; set; }

        public DateTime CreateDate { get; set; }
        public decimal? LastUpdateBy { get; set; }
        public DateTime LastUpdateDate { get; set; }
    }
}
